core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -4890,6 +4890,39 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-nutanix-qe-dis
     namespace: ci
+- from:
+    .awscred:
+      field: .awscred
+      item: jenkins-ci-iam
+    insights-live.yaml:
+      field: insights-live.yaml
+      item: insights-ci-account
+    pull-secret:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_app.ci_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_field: auth
+        email_field: email
+        item: cloud.openshift.com-pull-secret
+        registry_url: cloud.openshift.com
+      - auth_field: auth
+        email_field: email
+        item: quay.io-pull-secret
+        registry_url: quay.io
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+  to:
+  - cluster_groups:
```

Comment:
I think we can append this to https://github.com/openshift/release/blob/bfa728fb645d5473dbcf84f3d3343ab6b7cb4b7d/core-services/ci-secret-bootstrap/_config.yaml#L4855 together with nutanix-qe-dis like:
```
  to:
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe
    namespace: ci
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe-zone
    namespace: ci
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe-zone
    namespace: ci
```

Reply:
Fixed, pls review again, thanks.